admin/featured_categories.php

```diff
@@ -0,0 +1,550 @@
+      } else { // category id is valid
+        zen_redirect(zen_href_link(FILENAME_FEATURED_CATEGORIES, ($currentPage != 0 ? 'page=' . $currentPage . '&' : '') . 'action=new' . '&preID=' . (int)$_POST['pre_add_categories_id']));
+      }
+      break;
+  }
+}
+?>
+<!doctype html>
+<html <?php echo HTML_PARAMS; ?>>
+  <head>
+    <?php require DIR_WS_INCLUDES . 'admin_html_head.php'; ?>
+  </head>
+  <body>
+    <!-- header //-->
+    <?php require DIR_WS_INCLUDES . 'header.php'; ?>
+    <!-- header_eof //-->
+    <div class="container-fluid">
+      <!-- body //-->
+      <h1><?php echo HEADING_TITLE; ?></h1>
+      <!-- body_text //-->
+      <?php
+      if (($action === 'new') || ($action === 'edit')) {
+        $form_action = 'insert';
+        if (($action === 'edit') && isset($_GET['fID'])) {//update existing Featured
+          $form_action = 'update';
+
+          $category = $db->Execute("SELECT p.categories_id, pd.categories_name,
+                                          f.expires_date, f.featured_date_available
+                                   FROM " . TABLE_CATEGORIES . " p,
+                                        " . TABLE_CATEGORIES_DESCRIPTION . " pd,
+                                        " . TABLE_FEATURED_CATEGORIES . " f
+                                   WHERE p.categories_id = pd.categories_id
+                                   AND pd.language_id = " . (int)$_SESSION['languages_id'] . "
+                                   AND p.categories_id = f.categories_id
+                                   AND f.categories_id = " . (int)$_GET['fID']);
+
+          $fInfo = new objectInfo($category->fields);
+        } elseif (($action === 'new') && isset($_GET['preID'])) { //update existing Featured
+          $form_action = 'insert';
+
+          $category = $db->Execute("SELECT p.categories_id, pd.categories_name
+                                   FROM " . TABLE_CATEGORIES . " p,
+                                        " . TABLE_CATEGORIES_DESCRIPTION . " pd
+                                   WHERE p.categories_id = pd.categories_id
+                                   AND pd.language_id = " . (int)$_SESSION['languages_id'] . "
+                                   AND p.categories_id = " . (int)$_GET['preID']);
+
+          $fInfo = new objectInfo($category->fields);
+        } elseif (empty($_GET['preID'])) { // insert by category select dropdown
+          $fInfo = new objectInfo([]);
+
+// create an array of featured categories, which will be excluded from the pull down menu of categories
+// (when creating a new featured category)
+          $featured_array = [];
+          $featureds = $db->Execute("SELECT p.categories_id
+                                     FROM " . TABLE_CATEGORIES . " p,
+                                          " . TABLE_FEATURED_CATEGORIES . " f
+                                     WHERE f.categories_id = p.categories_id");
```

Comment:
Use `c` and `cd` as aliases, not `p` and `pd`, these are categories tables, not products.